drivers/note/note_driver.c

```diff
@@ -223,12 +223,14 @@ static void note_common(FAR struct tcb_s *tcb,
                         FAR struct note_common_s *note,
                         uint8_t length, uint8_t type)
 {
-#ifdef CONFIG_SCHED_INSTRUMENTATION_HIRES
   struct timespec ts;
-
+#ifdef CONFIG_SCHED_INSTRUMENTATION_HIRES
   clock_systime_timespec(&ts);
 #else
   clock_t systime = clock_systime_ticks();
+#  ifdef CONFIG_SCHED_INSTRUMENTATION_PERFCOUNT
+  uint32_t count = up_perf_gettime();
```

Comment:
why call both clock_systime_ticks  and up_perf_gettime?

Reply:
done